package-lock.json

```diff
@@ -2983,11 +2983,6 @@
         "@types/node": "*"
       }
     },
-    "@types/validator": {
```

Comment:
Should this one be removed?

Reply:
It has, hasn't it?

Reply:
Ah, you mean *should* it have been!

Reply:
Assuming later version of class-validator didn't need it - but I'll check.